lib/constants/error_messages.js

```diff
@@ -47,6 +47,7 @@ exports[404022] = 'Invalid region. The specified value must be a string.';
 exports[404023] = 'Invalid clientSessionKeepAlive. The specified value must be a boolean.';
 exports[404024] = 'Invalid clientSessionKeepAliveHeartbeatFrequency. The specified value must be a number.';
 exports[404025] = 'Invalid jsTreatIntegerAsBigInt. The specified value must be a boolean';
+exports[404026] = 'Invalid validateDefaultParameters. The specified value must be boolean';
```

Comment:
As we discussed in the previous meeting, you don't need to add a error code for NodeJS but let the GS error arise.

Reply:
Sure, will get that removed.